server/audit_math/supersimple.py

```diff
@@ -15,6 +15,17 @@
 o2: float = 0.0001
 u2: float = 0.0001
 
+# { ballot_id: { contest_id: { choice_id: 0 | 1 }}}
+CVRS = Dict[str, Dict[str, Dict[str, int]]]
+CVR = Dict[str, Dict[str, int]]
+
+
+class Discrepancy(TypedDict):
+    counted_as: int
```

Comment:
maybe name this `discrepancy` or `simple_error`?

Reply:
Yeah, I wasn't sure if I should follow the class naming convention or not.

Reply:
what's the class naming convention?

Reply:
If it's a class, it gets a capital first letter.

Reply:
oh yeah def follow that. i was referring to changing the name of `counted_as`